cardinal/world.go

```diff
@@ -123,13 +123,13 @@ func NewWorld(opts ...WorldOption) (*World, error) {
 		serverOptions: serverOptions,
 
 		// Core modules
-		worldStage:       worldstage.NewManager(),
-		MessageManager:   newMessageManager(),
-		SystemManager:    newSystemManager(),
-		componentManager: component.NewManager(&redisMetaStore),
-		QueryManager:     newQueryManager(),
-		router:           nil, // Will be set if run mode is production or its injected via options
-		txPool:           txpool.New(),
+		worldStage:     worldstage.NewManager(),
+		MessageManager: newMessageManager(),
+		SystemManager:  newSystemManager(),
+		Manager:        component.NewManager(&redisMetaStore),
```

Comment:
this should be ComponentManager

Reply:
It's changed. and I added a linter exception so that the linter will ignore the stuttering.